web.go

```diff
@@ -28,6 +30,7 @@ type WebProvider struct {
 	KeyFile  string `description:"SSL certificate"`
 	ReadOnly bool   `description:"Enable read only API"`
 	server   *Server
+	Auth     *types.Auth `description:"Auth Crendetials for Web Point"`
```

Comment:
Looks like I missed that. I guess, removing it and assume TOML file is the only way is fine. Thanks for pointing that out!

However, let's wait for @emilevauge :)